usr/share/rear/prep/NETFS/GNU/Linux/205_inspect_tar_capabilities.sh

```diff
@@ -0,0 +1,31 @@
+# prep/NETFS/GNU/Linux/205_inspect_tar_capabilities.sh
+# The purpose is to inspect the 'tar --usage' for certain capabilities and if found
+# add these to the BACKUP_PROG_OPTIONS array
+# See also issue https://github.com/rear/rear/issues/1175
+
+# As we will only inspect 'tar' we make one big if block:
+
+if [[ "$(basename $BACKUP_PROG)" = "tar" ]] ; then
+    # Verify extended attributes being present:
+    if tar --usage | grep -q "\-\-xattrs" ; then
+        BACKUP_PROG_OPTIONS=( "${BACKUP_PROG_OPTIONS[@]}" "--xattrs" )
+        PROGS=( "${PROGS[@]}" getfattr setfattr )
```

Comment:
Does `tar` require the `setfattr` programs to restore the archive with xattrs? Or this this more for the convenience of the user?

Reply:
@schlomo well it is always better to be as complete as possible. You never know.